recipes/spyder/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = "2.3.9" %}
+
+package:
+  name: spyder
+  version: {{ version }}
+
+source:
+  fn:  spyder-{{ version }}.zip
+  url: https://pypi.io/packages/source/s/spyder/spyder-{{ version }}.zip
+  md5: dd01e07a77123c128ff79ba57b97c1d7
+
+build:
+  number: 0
+  osx_is_app: True
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+    - pyqt
+    - rope
+    - pyflakes
+    - jedi  >=0.8
+    - qtconsole
+    - nbconvert
+    - pygments
+    - sphinx
+    - pep8
+    - psutil
+    - python.app  # [osx]
+
+test:
+  commands:
+    - USER=test spyder -h  # [unix]
+    - spyder -h  # [win]
+  imports:
+    - spyderlib
+
+app:
+  entry: spyder
+  icon: logo.png
+  summary: Scientific Python Development Environment
+  type: desk
+
+about:
+  home: https://github.com/spyder-ide/spyder
+  license: MIT
+  license_file: LICENSE
+  summary: Scientific PYthon Development EnviRonment
+  description: |
+    Spyder is a powerful interactive development environment for the Python
+    language with advanced editing, interactive testing, debugging and
+    introspection features. Additionally, Spyder is a numerical computing
+    environment thanks to the support of IPython and popular Python libraries
+    such as NumPy, SciPy, or matplotlib.
+  doc_url: http://pythonhosted.org/spyder/
+  dev_url: https://github.com/spyder-ide/spyder
+
+extra:
+  recipe-maintainers:
+    - ccordoba12
+    - goanpeca
```

Comment:
And add me to maintainers if you want.  I think the png is fine.  It goes with the menu entry.

+1 for merge.